media/css/firefox/features.less

```diff
@@ -0,0 +1,205 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+.fxfeatures-sprite(@posy) {
+    .at2x('/media/img/firefox/features/features-sprite.png', 129px, 214px);
+    background-position: 0 @posy;
+    background-repeat: no-repeat;
+    content: '';
+    display: block;
+    margin: @baseLine auto 0;
+    overflow: hidden;
+    width: 129px;
+}
+
+.mozid {
+    #outer-wrapper {
+        .aurora-bars;
+    }
+
+    #wrapper {
+        overflow: hidden;
+        padding: 0;
+        width: 100%;
+    }
+}
+
+#wrapper {
+    margin-top: 50px;
+}
+
+#main-feature {
+    color: #fff;
+
+    h1 {
+        .font-size(4rem);
+        color: #fff;
+        line-height: 1.2;
+        text-shadow: none;
+
+        @media (max-width: @breakDesktop) {
+            .font-size(2.625rem);
+        }
+
+        @media (max-width: @breakTablet) {
+            .font-size(2.25rem);
+        }
+    }
+
+    p {
+        .font-size(1.75rem);
+        .open-sans-light();
+        line-height: 1.4;
+
+        @media (max-width: @breakDesktop) {
+            .font-size(1.125rem);
+        }
+
+        @media (max-width: @breakTablet) {
+            .font-size(1rem);
+        }
+    }
+
+    h1,
+    p {
+        .span-all();
+        margin-bottom: @baseLine / 2;
+        text-align: center;
+    }
+}
+
+#fxfeature-list {
+    .clearfix();
+    list-style-type: none;
+    margin: 0 auto;
+    padding: 0;
+    width: @breakDesktop - (@baseLine * 2);
```

Comment:
For grid-related widths we should probably use `@gridGutterWidth` rather than `@baseLine` (which is type-related so better for vertical spacing). They both happen to be 20px so the result is the same, but the point of making these variables is that they could change. If we someday decided to change `@baseLine` to 24px the grid might break.